### PR TITLE
Base crate docker image on CentOS 7 and use JDK11

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,21 +4,23 @@
...
-        python3 \
-        openssl \
-        curl \
-        coreutils \
-    && apk add --no-cache --virtual .build-deps \
-        gnupg \
-        tar \
+RUN yum install -y yum-utils https://centos7.iuscommunity.org/ius-release.rpm \
+    && yum makecache \
+    && yum install -y python36u openssl java-1.8.0-openjdk \
```
Comment (mfussenegger) [inline Dockerfile]: java-1.8.0 can probably be dropped?

Comment (andreidan) [reply Dockerfile]: ah yes, of course, thanks

File: Dockerfile.j2 [hunk 2]
```diff
@@ -4,21 +4,23 @@
...
 #
 
-FROM alpine:3.8
+FROM centos:7
 
-RUN addgroup -g 1000 -S crate \
-    && adduser -u 1000 -G crate -h /crate -S crate
+RUN groupadd crate && useradd -u 1000 -g crate -d /crate crate
+
+RUN curl --retry 8 -s {{ JDK_URL }} | tar -C /opt -zxf -
```
Comment (autophagy) [inline Dockerfile.j2]: Would it be possible, after downloading the JDK, to verify it via it's sha526 signature (i.e https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz.sha256) before you extract it?

Comment (mfussenegger) [reply Dockerfile.j2]: Not sure if there is a point in the verification if we retrieve the checksum from the same location? 
If java.net is compromised both files can be replaced. Curl/http is TCP based - so corruption due to transfers is not likely.

Comment (autophagy) [reply Dockerfile.j2]: Acht, yeah, good point.

**Pull request merged**